puppet-ghostbuster.gemspec

```
@@ -12,6 +12,8 @@ Gem::Specification.new do |s|
   s.description = 'Try and find dead code in Puppet receipts'
   s.licenses    = 'Apache-2.0'
 
+  s.required_ruby_version = Gem::Requirement.new('>= 2.7')
```

```suggestion
  s.required_ruby_version = '>= 2.7'
```

isn't that equivalent? and easier to read :)

---

```
irb(main):002:0> Gem::Requirement.new('>= 2.7').to_s
=> ">= 2.7"
```

---

I'm fine with both. I copied it from the beaker gem.